### PR TITLE
Unnecessary local variables. Inline variable.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: maven-scm-providers/maven-scm-provider-hg/src/main/java/org/apache/maven/scm/provider/hg/HgScmProvider.java [hunk 1]
```diff
@@ -150,7 +148,7 @@ else if ( url.startsWith( "http" ) )
 
         }
 
-        result.repository = new HgScmProviderRepository( url );
+        result.repository = new HgScmProviderRepository(scmSpecificUrl);
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-hg/src/main/java/org/apache/maven/scm/provider/hg/HgScmProvider.java:151]: Spaces lost

File: maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java [hunk 2]
```diff
@@ -82,20 +82,19 @@ protected ScmResult executeEditCommand( ScmProviderRepository repository, ScmFil
             }
             for ( File f : fileSet.getFileList() )
             {
-                File dest = f;
                 File source = new File( sourcePath, SynergyUtil.removePrefix( fileSet.getBasedir(), f ) );
                 List<File> list = new LinkedList<File>();
                 list.add( source );
                 SynergyUtil.checkoutFiles( getLogger(), list, ccmAddr );
-                if ( !source.equals( dest ) )
+                if ( !source.equals(  f  ) )
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java:89]: Too much spaces

File: maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java [hunk 3]
```diff
@@ -82,20 +82,19 @@ protected ScmResult executeEditCommand( ScmProviderRepository repository, ScmFil
...
                 {
                     if ( getLogger().isDebugEnabled() )
                     {
-                        getLogger().debug( "Copy file [" + source + "] to expected folder [" + dest + "]." );
+                        getLogger().debug( "Copy file [" + source + "] to expected folder [" + f + "]." );
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java:93]: folder => directory

File: maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java [hunk 4]
```diff
@@ -82,20 +82,19 @@ protected ScmResult executeEditCommand( ScmProviderRepository repository, ScmFil
             }
             for ( File f : fileSet.getFileList() )
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java:83]: Why not rename `f` to `dest` directly?

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by arturobernalg** [comment]
Hi @michael-o 
Solved.
TY

File: maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java (hunk 4 above)
Comment (michael-o) [reply maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java:83]: What about this one?

Comment (arturobernalg) [reply maven-scm-providers/maven-scm-provider-synergy/src/main/java/org/apache/maven/scm/provider/synergy/command/edit/SynergyEditCommand.java]: Yep. Changed.
TY

**Review by michael-o — APPROVED** [review]

**Pull request closed without merging**